docs/table-of-contents.json

```diff
@@ -129,8 +130,8 @@
           "entries": [
             {"entry": "docs/api-reference/view"},
             {"entry": "docs/api-reference/map-view"},
+            {"entry": "docs/api-reference/globe-view"},
             {"entry": "docs/api-reference/first-person-view"},
-            {"entry": "docs/api-reference/third-person-view"},
```

Comment:
Weird to cut this entry here in this PR - especially without any other cut.

Reply:
This class/markdown file no longer exists.